submission/distributor.go

```diff
@@ -238,5 +257,7 @@ func NewDistributor(ll *loglist.LogList, plc ctpolicy.CTPolicy, lcBuilder LogCli
 		}
 		d.logClients[log.URL] = lc
 	}
+	d.mf = mf
```

Comment:
If you want to be friendly to users you could cope with a missing `MetricFactory`:
```golang
if mf == nil {
    mf = monitoring.InertMetricFactory{}
}
```
(Whether it's worthwhile depends on how many callers you're expecting)

Reply:
Added.